spki/Cargo.toml

```diff
@@ -21,13 +21,14 @@ der = { version = "0.7.2", features = ["oid"] }
 arbitrary = { version = "1.2", features = ["derive"], optional = true }
 base64ct = { version = "1", optional = true, default-features = false }
 sha2 = { version = "0.10", optional = true, default-features = false }
+signature = { version = "2.1.0", optional = true }
 
 [dev-dependencies]
 hex-literal = "0.4"
 tempfile = "3"
 
 [features]
-alloc = ["base64ct?/alloc", "der/alloc"]
+alloc = ["base64ct?/alloc", "der/alloc", "dep:signature"]
```

Comment:
Why is it gated on `alloc`?

Perhaps it should be a separate feature?

Reply:
well, `alloc` is required because we rely on `BitString` which holds the buffer in the heap. I can push a new feature flag but it would require `alloc` and come "free" with alloc.

Reply:
That's fine

Reply:
Although it seems like with f77783f you no longer need a `signature` dependency